pkgs/development/python-modules/scikit-network/default.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  cython,
+  setuptools,
+  numpy,
+  scipy,
+  pytest-runner,
+  pytest,
+  nose,
+  pluggy,
+}:
+buildPythonPackage rec {
+  pname = "scikit-network";
+  version = "0.33.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "scikit-network";
+    hash = "sha256-9/fMAutCZ7Fqk4P2RiYVLlm6Q11td7GjVmW2/BvPVy0=";
+  };
+
+  # can be removed if https://github.com/sknetwork-team/scikit-network/pull/581 is merged
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail "__builtins__.__NUMPY_SETUP__ = False" "__builtins__['__NUMPY_SETUP__'] = False"
+  '';
+
+  build-system = [
+    cython
+    setuptools
+    numpy
+    scipy
+    pytest-runner
+  ];
+
+  dependencies = [
+    numpy
+    scipy
+  ];
+
+  nativeCheckInputs = [
+    pytest
+    nose
+    pluggy
+  ];
+
+  pythonImportscheck = [ "sknetwork" ];
```

```suggestion
  pythonImportsCheck = [ "sknetwork" ];
```

Also, looking at e.g. scikit-learn-extra, it seems to import-check all cythonized modules .. so maybe we should do that here? Or does `import sknetwork` load all of them anyway? Not sure tho